README.md

```diff
@@ -15,39 +15,257 @@
+
+The downside to that task is that it causes the cleanup script to run _every_ day, even on the weekends when there is no activity.
+We can instead use a weekly trigger to fix this:
+
+~~~puppet
+scheduled_task { 'Disk Cleanup Weekly Repeating':
+  ensure    => 'present',
+  command   => 'C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe',
+  arguments => '-File "C:\\Scripts\\Clear-DiskSpace.ps1",
+  enabled   => 'true',
+  trigger   => [{
+    'schedule'         => 'weekly',
+    'start_time'       => '07:00',
+    'day_of_week'      => ['mon', 'tues', 'wed', 'thu', 'fri'], # Note the absence of Sunday and Monday
+    'minutes_interval' => '60',
+    'minutes_duration' => '720'
+  }],
+  user      => 'system',
+}
+~~~
+
+Similarly, we can schedule our cleanup script to run monthly if we decide the cleanup script doesn't need to run as often or is particularly resource intensive.
+The following example sets the scheduled task to run at 0700 on the first day of the month every month:
+
+~~~puppet
+scheduled_task { 'Disk Cleanup Monthly First Day':
+  ensure    => 'present',
+  command   => 'C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe',
+  arguments => '-File "C:\\Scripts\\Clear-DiskSpace.ps1",
+  enabled   => 'true',
+  trigger   => [{
+    'schedule'         => 'monthly',
+    'start_time'       => '07:00',
+    'on'               => [1]        # Run every month on the first day of the month.
+  }],
+  user      => 'system',
+}
+~~~
+
+With the monthly trigger above there is no guarantee that the first day of the month is on a weekend.
+This means that there's a reasonable chance that the script will execute during working hours and impact productivity.
+We can specify the trigger to run on the task on the _first saturday_ of the month instead of the first day:
+
+~~~puppet
+scheduled_task { 'Disk Cleanup Monthly First Saturday':
+  ensure    => 'present',
+  command   => 'C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe',
+  arguments => '-File "C:\\Scripts\\Clear-DiskSpace.ps1",
+  enabled   => 'true',
+  trigger   => [{
+    'schedule'       => 'monthly',
+    'start_time'     => '07:00',
+    'day_of_week'    => 'sat',     # Specify the day of the week to trigger on
+    'which_occurence => 'first'    # Specify which occurance to trigger on, up to fifth
+  }],
+  user      => 'system',
+}
+~~~
 
 ## Reference
 
 ### Provider
 
+* win32_taskscheduler: This provider manages scheduled tasks on Windows using the legacy API.
 * taskscheduler_api2: Adapts the Puppet scheduled_task resource to use the modern Version 2 API.
 
 ### Type
 
-* scheduled_task: See the [Puppet resource documentation](https://puppet.com/docs/puppet/latest/types/scheduled_task.html) for more information.
+#### scheduled_task
+
+Installs and manages Windows Scheduled Tasks.
+All attributes except `name`, `command`, and `trigger` are optional; see the description of the [`trigger`]() attribute for details on setting schedules.
```

Comment:
It doesn't appear that this link is working properly....

Reply:
Derp, completely missed this!